docs/asciidoc/back_matter.adoc

```diff
@@ -39,3 +37,111 @@ This spec defines a standard **Data Repository Service (DRS) API** (“the yello
+
+We use a DRS URI scheme rather than https://en.wikipedia.org/wiki/CURIE[Compact URIs (CURIEs)] directly since we feel that systems consuming DRS objects will be able to better differentiate a DRS URI.  CURIEs are widely used in the research community and we feel the fact that they can point to a wide variety of entities (HTML documents, PDFs, identities in data models, etc) makes it more difficult for systems to unambiguously identify entities as DRS objects.
+
+Still, to make compact identifiers work in DRS URIs we leverage the CURIE format used by identifiers.org/n2t.net.  Compact identifiers have the form:
+
+    prefix:accession
+
+The prefix can be divided into a `provider_code` (optional) and `namespace`.  The `accession` here is an Ark, DOI, Data GUID, or another issuers's local ID for the object being pointed to:
+
+    [provider_code/]namespace:accession
+
+Both the `provider_code` and `namespace` disallow spaces or punctuation, only lowercase alphanumerical characters, underscores and dots are allowed.
+
+https://n2t.net/e/compact_ids.html[Examples] include (from n2t.net):
+
+		PDB:2gc4
+		Taxon:9606
+		DOI:10.5281/ZENODO.1289856
+		ark:/47881/m6g15z54
+		IGSN:SSH000SUA
+
+TIP: DRS URIs using compact identifiers with resolvers registered in identifiers.org/n2t.net can be distinguished from the hostname-based DRS URIs below based on the required ":" which is not allowed in hostname-based URI.
+
+TIP: The CURIE format used by identifiers.org/n2t.net does not percent-encode reserved URI characters but, instead, relies on the first ":" character to separate prefix from accession. Since these accessions can contain any characters, and characters like "/" will interfere with DRS API calls, you _must_ percent encode the accessions extracted from DRS compact identifier-based URIs when using as DRS IDs in DRS GET requests.  For more information see the Note "DRS Client Compact Identifier-Based URI Resolution Process".
+
+See the documentation on https://n2t.net/e/compact_ids.html[n2t.net] and https://docs.identifiers.org/[identifiers.org] for much more information on the compact identifiers used there and details about the resolution process.
+
+.Note: DRS Client Compact Identifier-Based URI Resolution Process
+****
+A DRS client identifies the a DRS URI compact identifier components using the first occurance of "/" (optional) and ":" characters. These are not allowed inside the provider_code (optional) or the namespace. The ":" character is not allowed in a Hostname-based DRS URI, providing a convenient mechanism to differentiate them. Once the provider_code (optional) and namespace are extracted from a DRS compact identifier-based URI, a client can use services on identifiers.org to identify available resolvers.
```

Comment:
> A DRS client identifies the a DRS URI compact identifier components using the first occurance of "/" (optional) and ":" characters.

I'm a little bit lost on the first occurrence of `/` since it can be interpreted to be the `/` in `drs://`. So a DRS client knows that it is a compact identifier-based URI by optional `/` after the provider code and the `:`. It might be clearer to just emphasize that the indication of `:` in the DRS ID will signal a compact identifier-based URI if it will always exist

Reply:
The spec has been reorganized but this is what the text currently says:

1) The client parses the string to extract the prefix of `drs.42` and the accession of `314159`, using the first occurrence of a colon (":") character after the initial `drs://` as a delimiter. (The colon character is not allowed in a Hostname-based DRS URI, making it easy to tell them apart.)

I think that better matches what you're proposing here.